tests/dbfs/test_api.py

```diff
@@ -133,14 +133,9 @@ def test_put_file(self, dbfs_api, tmpdir):
         api_mock = dbfs_api.client
         test_handle = 0
         api_mock.create.return_value = {'handle': test_handle}
+        # Should succeed.
```

Comment:
This comment isn't really helpful.
Can you assert that it succedded by using other API's like list and matching the content?

Reply:
I don't think this is possible since the API is a mock. It would only be possible to do this if I explicitly define what the return value of other APIs are which does not help us for testing. Please, correct me if I am wrong.